src/Symfony/Component/Validator/Constraints/Bic.php

```diff
@@ -52,8 +52,8 @@ public function __construct($options = null)
             @trigger_error(sprintf('Using the "%s" constraint without the "symfony/intl" component installed is deprecated since Symfony 4.2.', __CLASS__), E_USER_DEPRECATED);
         }
 
-        if (isset($options['iban']) && isset($options['ibanPropertyPath'])) {
-            throw new ConstraintDefinitionException(sprintf('The "iban" and "ibanPropertyPath" options of the Iban constraint cannot be used at the same time.', self::class));
+        if (isset($options['iban'], $options['ibanPropertyPath'])) {
```

Comment:
This change should be reverted.

Reply:
Because I am interested, can you explain why?

Reply:
@xabbuh can you please elaborate? I'm also interested :)

Reply:
because it requires more knowledge about a syntax that is less common
the previous CS is obvious, this one is less obvious.